class-bios/daniel-rendall.md

```diff
@@ -0,0 +1,13 @@
+# Daniel Rendall
+![Picture of Daniel]()
```

Comment:
No picture? :(

Reply:
if you want to add one later, please make a new Pull Request. I'll approve for now.